### DISCUSSION TITLE
Envelope for new messages

### DISCUSSION THREAD

**Discussion by wahey** [BimmerGestalt/AAIdrive#835] [Q&A]
I own a 2017 F46 Grantourer and started to experiment with AAIdrive (it had a different name at that time) years back but never managed to get Spotify working. But at least I got my mails and whatsapp messages forwared to the iDrive and was notified by a small envelope on the display and could then read the messages via "Office -> Messages (Nachrichten)". All this via Blutooth, USB never worked.
I got a new cellphone now (Honor 200 Lite), installed AAIdrive and myBMW, connected it via USB to the car and VOILA, I have full Spotify in my car for the first time. THIS IS AMAZING!!!!! thank you, thank you, thank you, nearly feels like having a new car. Now my question: what do i need to do to get even the messages forwarded again just like it worked before, getting notified with that small envelope if new messages come in and then read them it doesn't matter where? Would be great if that would be possible

**Comment by hufman** [comment]
AAIdrive's Notifications support should work the same on different phones. Check the Notifications tab in AAIdrive's phone interface to check if any permissions need to be granted.
Also note that the Envelope icon is part of the car's Office interface. AAIdrive's own Notifications interface is hiding as the unlabeled Book icon in the Phone Apps section of the ConnectedDrive menu. This menu will show you the full list of notifications on the phone (not just any new ones that have arrived since plugging into the car), quick access for toggling some of the popup settings, and allow for selecting notification actions or replying to messages.
Spotify should work better over Bluetooth, I've had terrible luck with USB connection reliability and USB also can't enable the Media button to open the Spotify app. But if that works better for you, great!
